src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -897,10 +896,11 @@ $kCodeBlockDarkLineMatchBackgroundColor: rgba(255, 255, 255, 0.12); // we don't
 
     .code-block-secondary-actions {
       display: flex;
+      float: right;
       gap: var(--kui-space-40, $kui-space-40);
       margin-right: var(--kui-space-40, $kui-space-40);
       margin-top: var(--kui-space-40, $kui-space-40);
```

Comment:
I found that if we remove these two lines, the copy button will fit perfectly in the right place. I checked the Figma component and it seems the spacing is correct.

<img width="260" alt="image" src="https://github.com/user-attachments/assets/4e3ff09f-2c53-44f5-8d73-a73a28f3fa92" />

Reply:
Oh I found that this margin is also used by the single line version, maybe we should keep them for that.

Reply:
<img width="830" alt="Screenshot 2025-01-15 at 5 17 59 PM" src="https://github.com/user-attachments/assets/2c1bda04-ce7f-474e-a5aa-0410a2c499d7" />
It looks fine in the single line mode without the margins

Reply:
👌